main/daily_update.py

```diff
@@ -268,7 +272,15 @@ def main():
         print('Sql Server connected')
     except Exception as e:
         print(str(e))
+# ------------------------- Check if the HealthPro data is well loaded ------------------------------------------------#
 
+    contact_amount = extract_data(engine_sf, 'sf_contact_update.sql').shape[0]
+    while contact_amount > 1000:
+        print('The HealthPro API is still working')
+        time.sleep(600)
+        contact_amount = extract_data(engine_sf, 'sf_contact_update.sql').shape[0]
+    else:
+        print('HealthPro data is well loaded')
 # ------------------------- Extract contacts and journeys from Salesforce and save in local DB ------------------------#
```

Comment:
Healthpro is ok if shape / length > 100?

Reply:
I think 100 and 1000 are both fine.